### PR TITLE
VTCCA-4798 comma format assessments rateable values

### PR DESCRIPTION
**Pull request by DanSmith98**

### CODE REVIEW

File: app/utils/Formatters.scala [hunk 1]
```diff
@@ -58,4 +59,18 @@ object Formatters {
...
     value match { case Some(paramValue) if paramValue != "" => s"&$name=${paramValue.toUpperCase.trim}"; case _ => "" }
 
+  def formatCurrency[A: Numeric](total: A): String = {
+    val currencyFormatter = getCurrencyInstance(java.util.Locale.UK)
+    currencyFormatter.setRoundingMode(java.math.RoundingMode.DOWN)
+
+    currencyFormatter.format(total)
+  }
+
+  def formatCurrencyRoundedToPounds[A: Numeric](total: A): String = {
```
Comment (PeterPerhac) [inline app/utils/Formatters.scala:69]: could we add a `maximumFractionDigits:Option[Int] = None` parameter to `formatCurrency` and use it like so:

```scala
maximumFractionDigits.foreach(digits => currencyFormatter.setMaximumFractionDigits(digits))
```

then the `formatCurrencyRoundedToPounds` method would pass-through to it, like so `formatCurrency(total, Some(0))`

**Comment by PeterPerhac** [comment]
This looks good, I just find that the two currency formatting methods are largely similar, with only 1 additional line in one of them. I would like you to express the second one in terms of the first one.

File: app/utils/Formatters.scala (hunk 1 above)
Comment (DanSmith98) [reply app/utils/Formatters.scala:69]: I like it, updating now

**Comment by PeterPerhac** [comment]
👍  looks good to me

**Comment by habeeb01** [comment]
LGTM 👍

**Pull request merged**